backend/resources/org/akvo/dash/migrations_tenants/009-dashboard.up.sql

```diff
@@ -15,18 +15,19 @@ END$$;
 --;;
 
 CREATE TABLE dashboard_visualisation (
+    id text DEFAULT uuid_generate_v4()::text UNIQUE NOT NULL,
```

Comment:
I think `id` this should be the `PRIMARY KEY`, and have a `UNIQUE` constraint in the combination of `(dashboard_id, visualisation_id)`